examples/acm/certificate.yaml

```diff
@@ -8,13 +8,32 @@ metadata:
   name: example-dns
   labels:
     testing.upbound.io/example-name: certificate
+  annotations:
+    upjet.upbound.io/manual-intervention: "It requires a valid certificate and privateKeySecret"
 spec:
   forProvider:
     region: us-west-1
-    domainName: example-dns.upbound-providers.io
-    tags:
-      Environment: test
-    validationMethod: DNS
```

Comment:
Why did we remove these arguments?

Reply:
It was deleted to reproduce bug and check bugfix. According example in https://github.com/upbound/provider-aws/issues/464 I used provided example for this. I can back this code and delete my changes used for testing.